services/viva-ms/helpers/completions.js

```diff
@@ -76,6 +78,11 @@ async function getCaseOnWorkflowId(personalNumber, workflowId) {
   return caseItem;
 }
 
+function isCaseStateCompletions(caseItem) {
+  const { state } = caseItem;
+  return COMPLETIONS_TYPE.reduce((all, type) => state.includes(type) || all, false);
```

Comment:
wouldn't `some` do the same thing?

```js
COMPLETIONS_TYPE.some((value) => value.includes(state)))
```

Reply:
Will fix in the release branch because I have the tests in there. :)